### PR TITLE
Fix building guide link.

### PR DESCRIPTION
**Pull request by BezPowell**
Link to information on how to build for different platforms went to a 404. Changing to the current url.

### CODE REVIEW

**Comment by maxammann** [comment]
@Drabble do you still have a proper building.md documentation planned?

**Comment by Drabble** [comment]
> @Drabble do you still have a proper building.md documentation planned?

For now no. I think it was important to document how to run on any platform (for development purposes). However building libraries/demos for multiple environments will be done in the Github Actions. So it is probably more important to setup the Github Actions first, and the Actions can act as a documentation in case anyone wants to build locally. 

In my opinion we can merge this PR on the main branch, maybe just change the comment from "building" to "running" :)

File: README.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ cargo run -p maplibre-demo
 ```
 
 More information about building for different platforms can be
```
Comment (Drabble) [inline README.md:113]:
```suggestion
More information about running the demos on different platforms can be
```

**Review by maxammann — APPROVED** [review]

**Comment by maxammann** [comment]
Thanks all!

**Pull request merged**